example01/README.md

```diff
@@ -4,4 +4,4 @@ A simple (and ulgy) Task Manager written using `innerself`.
 
 To install the dependencies and serve the example locally run:
 
-    $ make
+    $ npm start
```

Comment:
Can you add `$ npm install` as the first step here, to account for the `To install the dependencies…` part of the sentence?